app/src/main/java/com/islamversity/reyan/service/NotificationDataType.kt

```diff
@@ -0,0 +1,17 @@
+package com.islamversity.reyan.service
+
+enum class NotificationDataType {
+
+    NONE, NEW_VERSION, FORCE_UPDATE;
+
+    companion object {
+
+        val notificationTypeMap = hashMapOf(
+            "new_version" to NEW_VERSION,
+            "force_update" to FORCE_UPDATE
+        )
```

Comment:
you can use a varaible inside `enum`

Reply:
what ???

Reply:
please remove the map
and use a `val` inside each enum type

Reply:
like  this ?
val NEW_VERSION = "new_version"